protocol/communities/manager.go

```diff
@@ -2948,6 +3023,22 @@ type CheckChannelViewAndPostPermissionsResult struct {
 	Permissions map[string]*PermissionTokenCriteriaResult `json:"permissions"`
 }
 
+func (m *Manager) computeViewOnlySatisfied(hasViewOnlyPermissions bool, hasViewAndPostPermissions bool, checkedViewOnlySatisfied bool, checkedViewAndPostSatisified bool) bool {
+	if (hasViewAndPostPermissions && !hasViewOnlyPermissions) || (hasViewOnlyPermissions && hasViewAndPostPermissions && checkedViewAndPostSatisified) {
+		return checkedViewAndPostSatisified
+	} else {
+		return checkedViewOnlySatisfied
+	}
+}
+
+func (m *Manager) computeViewAndPostSatisfied(hasViewOnlyPermissions bool, hasViewAndPostPermissions bool, checkedViewAndPostSatisified bool) bool {
+	if hasViewOnlyPermissions && !hasViewAndPostPermissions {
+		return false
```

Comment:
I see that this was the old code just moved to a new function, but I wonder if maybe we should track privilege levels here instead?
I guess it can be done later, but it would be good if status-go always returned the real value so clients don't have to check themselves.

Reply:
What do you mean by "privilege levels" ? These functions are used in both light and normal functions to keep consistency and the same results. Sure, we can try to "restructure" later, but can you elaborate?

Reply:
I mean that those will return if we satisfy according to the tokens and such, but I wonder if it takes into account that admins have access to everything. 
I guess in theory, with your new Light functions, admins will always be `satisfied: true`, so we'll basically never call the ones with the provider call, so maybe my suggestion doesn't make sense.
Anyway, like I said, it's not for now